apps/marketplace/components/Opportunities/OpportunitiesFilters.js

```diff
@@ -0,0 +1,465 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUaccordion from '@gov.au/accordion/lib/js/react.js'
+import { AUcheckbox } from '@gov.au/control-input/lib/js/react.js'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import styles from './Opportunities.scss'
+
+export class OpportunitiesFilters extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      accordionOpen: props.accordionOpen,
```

Comment:
I realize this is all "internal" state but I still feel like it should be managed in redux

Reply:
I just feel like you've had to do a lot of the boilerplate that we get for free using react-redux-form + redux.  The general pattern we've used in the rest of the components is to use a Form when user submitting.